### DISCUSSION TITLE
[FE] サイドバー

### DISCUSSION THREAD

**Discussion by Rinasham** [Hackathon5-TeamD/evolution-quest#50] [Ideas]
サイドバーにreact-burger-menuのライブラリ使うと楽そうですがどうでしょうか？(`·⊝·´)

ただ、このライブラリ使うとCSS-Modulesが当てられない箇所があるのでそこに関しては普通のCSSでスタイリングすることになってしまうんですが。。。

#49

**Comment by Terralien-jp** [comment]
セマンティックUIでスライドインメニューがあったような気がしますが、ハンバーガーメニューだと挙動違う感じです(?_?)

**Reply by Rinasham** [reply]
Semantic UIにあるんですね！知らなかったですw
そっちで大丈夫です〜！

**Reply by Terralien-jp** [reply]
前回スライドインメニュー作成する際に見たはずなのですが、どれだかわからなくなっている不具合です…ｗ

**Reply by Rinasham** [reply]
ちなみになんですが今ググってみてるんですけどどれがそうなのか分からなくて(汗)
どれでも大丈夫なのでこれだよ〜って分かるリンク教えてもらえると助かります(இдஇ; )ｽﾐﾏｾﾝ

**Reply by Terralien-jp** [reply]
https://react.semantic-ui.com/modules/sidebar/
これかな？
react-burger-menuのがリッチは間違いないですｗ

**Reply by Rinasham** [reply]
ありがとうございますー！
Semantic UI CSSとかSemantic UI Reactとかどれがどれなんだかよく分からない私ですどうも。。。笑
ドキュメントむずかし！！笑

**Reply by Rinasham** [reply]
書いてみてるんですが、Semantic UIのサイドバーが圧倒的に書きづらい件について。。。笑

**Comment by Terralien-jp** [comment]
私個人としてはやりやすい方でいいんじゃないかな？と思っていますｗ
それはそれで挑戦したことになりますし！

**Reply by Terralien-jp** [reply]
ツリーミスったｗ

**Reply by Rinasham** [reply]
やっぱりSemantic UIのサイドバー嫌です！！笑

**Reply by Terralien-jp** [reply]
代替案試してみましょう！
